backend/src/main/java/ca/bc/gov/hlth/hnweb/converter/rapid/RPBSPWB0Converter.java

```diff
@@ -0,0 +1,59 @@
+package ca.bc.gov.hlth.hnweb.converter.rapid;
+
+import org.apache.commons.lang3.StringUtils;
+
+import ca.bc.gov.hlth.hnweb.model.rapid.RPBSHeader;
+import ca.bc.gov.hlth.hnweb.model.rapid.RPBSPWB0;
+import ca.bc.gov.hlth.hnweb.model.rapid.WB0;
+import ca.bc.gov.hlth.hnweb.model.rest.groupmember.AddDependentRequest;
+import ca.bc.gov.hlth.hnweb.model.rest.groupmember.AddDependentResponse;
+
+public class RPBSPWB0Converter extends BaseRapidConverter {
+	
+	private static final String TRAN_CODE = "RPBSPWB0";
+
+	public RPBSPWB0Converter() {
+		super();
+	}
+	
+	public RPBSPWB0 convertRequest(AddDependentRequest request) {
+		RPBSHeader rpbsHeader = new RPBSHeader();
+		rpbsHeader.setOrganization(userInfo.getOrganization());
+		rpbsHeader.setTranCode(getTranCode());
+
+		WB0 wb0 = new WB0();
+		wb0.setGroupNumber(request.getGroupNumber());
+		wb0.setPhn(request.getPhn());
+		wb0.setBenficiaryPHN(request.getDependentPhn());
+		wb0.setCoverageEffectiveDate(formatDate(request.getCoverageEffectiveDate()));
+		wb0.setRelationshipCode(request.getRelationship());
+		wb0.setStudentFlag(request.getIsStudent());
+		if (StringUtils.equals("Y", request.getIsStudent())) {
+			wb0.setStudentEndDate(formatDate(request.getStudentEndDate()));
+		} else {
+			wb0.setStudentEndDate("");
+		}
```

Comment:
Could do this with a ternary operator. Your choice though.

Reply:
Don't think ternary would work well here as it's not resolving a value but calling methods that return void.

Reply:
I was thinking of a ternary within the setter. Again, not required.

Reply:
OK, will leave as is.